src/Microsoft.Dnx.DesignTimeHost/Models/OutgoingMessages/DependencyDescription.cs

```diff
@@ -22,6 +24,31 @@ public class DependencyDescription
 
         public IEnumerable<DependencyItem> Dependencies { get; set; }
 
+        public DiagnosticMessage Error { get; set; }
```

Comment:
Today, one library has at most one error or warning. Nevertheless, I'll make it an Array.